paasta_tools/kubernetes_tools.py

```diff
@@ -1000,6 +1004,16 @@ def get_secret_volume_name(self, secret_volume: SecretVolume) -> str:
             "secret--{name}".format(name=secret_volume["secret_name"]), length_limit=63
         )
 
+    def get_projected_volume_name(self, projected_volume: ProjectedVolume) -> str:
+        return self.get_sanitised_volume_name(
+            "projected-sa--{audiences}".format(
+                audiences="-".join(
+                    src["audience"] for src in projected_volume["sources"]
```

Comment:
i know that the schema supports multiple sources, but out of curiosity when / would we use this in practice?

Reply:
never, I mostly just translated the k8s configs 1:1, but it actually make sense to be opinionated and only allow single values, I'll fix that.